src/clj/clj_diff/core.clj

```diff
@@ -53,25 +61,39 @@
 
 (defmethod patch :default
   [s edit-script]
-  (patch* s edit-script))
+  (into []
+    (mapcat
+      (fn [item]
+        (cond
+          (and (map? item) (contains? item :clj-diff.core/delete)) nil
+          (and (map? item) (contains? item :clj-diff.core/insert)) (:clj-diff.core/insert item)
+          :else (list item))))
+    (patch* s edit-script)))
 
 (defmethod patch String
   [s edit-script]
-  (apply str (patch* s edit-script)))
+  (apply str
+    (mapcat
+      (fn [item]
+        (cond
+          (and (map? item) (contains? item :clj-diff.core/delete)) nil
+          (and (map? item) (contains? item :clj-diff.core/insert)) (:clj-diff.core/insert item)
+          :else (list item)))
+      (patch* s edit-script))))
```

Comment:
Is this the same as the default case? If so it might be worth adding a comment here pointing that out. Just for ease of reading in the future.

Reply:
the cases differ slightly